### PR TITLE
2023-08-09 release

### PR DESCRIPTION
**Pull request by aleksanderziel**
2023-08-09 release

### CODE REVIEW

File: newsletters/2023-08-09-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+* [ZK-Roller-Coaster #10](https://taiko.mirror.xyz/225yWAX3qFRNHSjNJ2gXGgkUbi-OGLhidBaNqbOW_Ds) - biweekly update on ZK world. (02.08.2023)
+* [What are zk-rollups? Solving Ethereum’s scalability issue](https://crypto.news/learn/what-are-zk-rollups-solving-ethereums-scalability-issue/) - article by Adrian Zmudzinski. (03.08.2023)
+* [Tamper Proofing in the Digital Age: A Look at Proof of SQL](https://hackernoon.com/tamper-proofing-in-the-digital-age-a-look-at-proof-of-sql) - article by Gabriel Mangalindan. (03.08.2023)
+* [SwiftRange: A Short and Efficient Zero-Knowledge Range Argument For Confidential Transactions and More](https://eprint.iacr.org/2023/1185.pdf) - academic text by Nan Wang, Sid Chi-Kin Chau and Dongxi Liu. (04.08.2023)
+* [How I learn Zero Knowledge](https://medium.com/coinmonks/how-i-learn-zero-knowledge-45a1afb7d789) - blog post by Lauri Peltonen. (06.08.2023)
+* [Resolving a Cairo 2 error building a Smart Contract : “internal error: entered unreachable code”](https://medium.com/@elielmathe/resolving-a-puzzling-cairo-2-syntax-error-in-smart-contracts-a70368480ec8) - tutorial by El Mathe. (06.08.2023)
```
Comment (zie1ony) [inline newsletters/2023-08-09-newsletter.md:15]: This is not interesting. Remove it.

File: newsletters/2023-08-09-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+* [A Primer for the Zero-Knowledge Cryptography: Part I](https://medium.com/@rabiafatima/a-primer-for-the-zero-knowledge-cryptography-part-i-85b5d5190aef) - article by Rabia Fatima. (06.08.2023)
```
Comment (zie1ony) [inline newsletters/2023-08-09-newsletter.md:16]: Top Pick

File: newsletters/2023-08-09-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+* [Scaling Ethereum with zkRollups: Empowering the Future of Blockchain Technology](https://www.deadlinenews.co.uk/2023/08/07/scaling-ethereum-with-zkrollups-empowering-the-future-of-blockchain-technology/) - article by guest_deadline. (07.08.2023)
```
Comment (zie1ony) [inline newsletters/2023-08-09-newsletter.md:17]: Remove it.

File: newsletters/2023-08-09-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+* [Endgame: Proof of Governance](https://dba.mirror.xyz/UTPfxWe65dYrUu_RJX-5VkAJypFRyw3AZh6m0dRXYZk) - article by Jon Charbonneau. (07.08.2023)
```
Comment (zie1ony) [inline newsletters/2023-08-09-newsletter.md:18]: Top Pick

File: newsletters/2023-08-09-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+### Twitter
+* [Diagram of how a zero knowledge proof system is constructed](https://twitter.com/iam_preethi/status/1687131541581762560). (03.08.2023)
+* [Cośtam](https://twitter.com/0xSachinK/status/1687816562202292224). (05.08.2023)
+* [Layer 2 Landscape #66](https://twitter.com/bitsplaining/status/1688397926685708288). (07.08.2023)
+
+### YouTube Clips
+* [Your Game Will Suck Without Zero Knowledge Proofs - Frank Chen, Aleo](https://www.youtube.com/watch?v=_G3lAUrbYIU) - lecture by Frank Chen. (02.08.2023)
+* [Interoperability/fragmentation of L2s and fractal L3s](https://www.youtube.com/watch?v=ThYpLCmS73k). (04.08.2023) 
+* [ZK Paris: Tarrance Van As - Applications of Zero Knowledge Primitives to Gaming](https://www.youtube.com/watch?v=b1F0W8Wy3kw) - lecture by Tarrance Van As. (04.08.2023)
```
Comment (zie1ony) [inline newsletters/2023-08-09-newsletter.md:40]: Top Pick

File: newsletters/2023-08-09-newsletter.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+* [LambdaClass And Starknet Foundation Going Forward](https://blog.lambdaclass.com/lambdaclass-starknet-foundation-going-forward/). (03.08.2023)
+* [Polyhydra is partnering with EigenLayer](https://twitter.com/PolyhedraZK/status/1687145905085128704). (03.08.2023)
+* [DRAMA! Polygon Zero accuses Matter Labs’ developers of plagiarism](https://cointelegraph.com/news/polygon-zero-accuses-matter-labs-developers-of-plagiarism). (03.08.2023)
+* [Scroll - Weekly Update](https://twitter.com/Scroll_ZKP/status/1687619860845735938). (05.08.2023)
+* [Mantle - The Arrival of Mantle Mainnet & Our Ecosystem](https://www.mantle.xyz/blog/community/mantle-mainnet-arrival-ecosystem-ama-recap). (07.08.2023)
+* [Honey, I Shrunk the Proof: Enabling on-chain verification for RISC Zero & Bonsai](https://www.risczero.com/news/on-chain-verification). (08.08.2023)
+
+### Twitter
+* [Diagram of how a zero knowledge proof system is constructed](https://twitter.com/iam_preethi/status/1687131541581762560). (03.08.2023)
+* [Cośtam](https://twitter.com/0xSachinK/status/1687816562202292224). (05.08.2023)
```
Comment (zie1ony) [inline newsletters/2023-08-09-newsletter.md:34]: Cośtam?

**Review by zie1ony — CHANGES_REQUESTED** [review]
Add all the whitepapers and articles I mentioned as TopPicks to README.md

File: newsletters/2023-08-09-newsletter.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+
+### ZK projects
+* [Get to Know Dusk: Our Network Architecture](https://dusk.network/news/get-to-know-dusk-our-network-architecture/). (02.08.2023)
+* [dYdX launches Testnet 2](https://dydx.exchange/blog/public-testnet-update). (03.08.2023)
+* [LambdaClass And Starknet Foundation Going Forward](https://blog.lambdaclass.com/lambdaclass-starknet-foundation-going-forward/). (03.08.2023)
+* [Polyhydra is partnering with EigenLayer](https://twitter.com/PolyhedraZK/status/1687145905085128704). (03.08.2023)
+* [DRAMA! Polygon Zero accuses Matter Labs’ developers of plagiarism](https://cointelegraph.com/news/polygon-zero-accuses-matter-labs-developers-of-plagiarism). (03.08.2023)
+* [Scroll - Weekly Update](https://twitter.com/Scroll_ZKP/status/1687619860845735938). (05.08.2023)
+* [Mantle - The Arrival of Mantle Mainnet & Our Ecosystem](https://www.mantle.xyz/blog/community/mantle-mainnet-arrival-ecosystem-ama-recap). (07.08.2023)
+* [Honey, I Shrunk the Proof: Enabling on-chain verification for RISC Zero & Bonsai](https://www.risczero.com/news/on-chain-verification). (08.08.2023)
```
Comment (kubaplas) [inline newsletters/2023-08-09-newsletter.md:30]: Awesome

File: newsletters/2023-08-23-newsletter.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+
+### Top picks
+* [How to code FRI from scratch](https://blog.lambdaclass.com/how-to-code-fri-from-scratch/) - tutorial by Not A Monad. (18.08.2023)
+* [Beyond ZK: Next Steps for Compliance and Constrained Encryption on Blockchains with Matthew D. Green](https://www.youtube.com/watch?v=UZDFhdYshcU) - lecture by Matthew D. Green. (22.08.2023)
+
+### New articles 
+* [ZK-Roller-Coaster #11](https://taiko.mirror.xyz/QHWZW8EJ9CM8jnbWEJyvULvpH6Pn2nHWfo93Z0UH21Y) - biweekly update on ZK world. (16.08.2023)
+* [Optimal and Seamless ZkRollups on Polkadot](https://medium.com/coinmonks/optimal-and-seamless-zkrollups-on-polkadot-b8ca1bf434f2) - article by Kogarashi Network. (16.08.2023)
+* [From Layer1 to zkEVM: Opside's ZK-Rollup LaunchBase Is Offering a Plethora of Choices](https://hackernoon.com/from-layer1-to-zkevm-opsides-zk-rollup-launchbase-is-offering-a-plethora-of-choices) - article by Opside. (16.08.2023)
+* [The Cross-Domain Thesis Part 1: Setting The Stage](https://mirror.xyz/0x59567ea31347ba1430939E8a8aC58fDa014aBed9/e2pIdHzlaBi6m5GhsRGK0g5DyjgCVuU4lE614XmES5s) - article by Maven11 Research. (17.08.2023)
```
Comment (zie1ony) [inline newsletters/2023-08-23-newsletter.md:12]: Top Pick

File: newsletters/2023-08-23-newsletter.md [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+* [How to code FRI from scratch](https://blog.lambdaclass.com/how-to-code-fri-from-scratch/) - tutorial by Not A Monad. (18.08.2023)
```
Comment (zie1ony) [inline newsletters/2023-08-23-newsletter.md:13]: Top Pick

File: newsletters/2023-08-23-newsletter.md [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+* [zkEVM: a marketing fad or a game changer?](https://medium.com/coinmonks/zkevm-a-marketing-fad-or-a-game-changer-282740d2136) - article by Hira Siddiqui. (19.08.2023)
+* [Deciphering how zkEVMs make Ethereum more scalable employing ZK proofs & Rollups](https://medium.com/coinmonks/deciphering-how-zkevms-make-ethereum-more-scalable-employing-zk-proofs-rollups-9e71024c4a92) - article by Kikctikcy. (19.08.2023)
+* [Multi-Party Computation and Zero-Knowledge Proofs: Ensuring Privacy in the Digital Age](https://web3illy.medium.com/multi-party-computation-and-zero-knowledge-proofs-ensuring-privacy-in-the-digital-age-15d65b10be5) - article by Illy’s Web3 blog. (21.08.2023) 
+* [How Bitcoin Usage Could Skyrocket Thanks To Zero-Knowledge Proofs](https://www.forbes.com/sites/digital-assets/2023/08/21/how-bitcoin-usage-could-skyrocket-thanks-to-zero-knowledge-proofs/) - article by David Tse. (21.08.2023)
+* [Batchman and Robin: Batched and Non-batched Branching for Interactive ZK](https://eprint.iacr.org/2023/1257.pdf) - academic text by Yibin Yang, David Heath, Carmit Hazay, Vladimir Kolesnikov and Muthuramakrishnan Venkitasubramaniam. (21.08.2023)
```
Comment (zie1ony) [inline newsletters/2023-08-23-newsletter.md:18]: Top Pick

File: newsletters/2023-08-23-newsletter.md [hunk 11]
```diff
@@ -0,0 +1,65 @@
...
+* [Noir Contract Syntax: Inside Aztec Labs](https://www.youtube.com/watch?v=CG61fyfVwmY) - webinar by Jake Fecher. (22.08.2023)
+* [The Kernel Circuit: Inside Aztec Labs](https://www.youtube.com/watch?v=6rTQFBYvOpA) - webinar by David Banks. (22.08.2023)
+* [Beyond ZK: Next Steps for Compliance and Constrained Encryption on Blockchains with Matthew D. Green](https://www.youtube.com/watch?v=UZDFhdYshcU) - lecture by Matthew D. Green. (22.08.2023)
+
+### Podcasts and other talks
+* Weekly [Zero Knowledge Podcast](https://zeroknowledge.fm/288-2/) episode - *Episode 288: Quantum Cryptography with Or Sattath*. (16.08.2023) 
+* [Epicenter Podcast](https://www.youtube.com/watch?v=BTeSHnVEluo/) episode - *Sandeep Nailwal: Polygon 2.0 - The New Value Layer of the Internet? Ep. 509*. (18.08.2023)  
+* [Protocol:Gemini + Loopring discussing the Future of Gaming + Web3 (full recording from xSpaces)](https://www.youtube.com/watch?v=DqfEBPXJtTk) - discussion by Loopring. (20.08.2023)
+
+### Upcoming events
```
Comment (zie1ony) [inline newsletters/2023-08-23-newsletter.md:54]: add https://zkwarsaw.dev/

File: newsletters/2023-08-23-newsletter.md [hunk 12]
```diff
@@ -0,0 +1,65 @@
...
+### Twitter
+* [The Funding Gap in Projects](https://twitter.com/ETH_Daily/status/1692211841110356208). (17.08.2023)
+* [Thread about privacy on Ethereum](https://twitter.com/ZKValidator/status/1692997435176448004). (19.08.2023)
+* [Thread about knowing that program is being verified](https://twitter.com/bkiepuszewski/status/1693289010020466940). (20.08.2023)
+
+### YouTube Clips
+* [Ethereum Storage Proofs in a ZK-SNARK, in the Browser](https://www.youtube.com/watch?v=GbiV6exphuw) - webinar by Joe Andrews. (16.08.2023)
+* [The Magic of Zero-Knowledge Proofs #SoME3](https://www.youtube.com/watch?v=FfeXX6OLq8w) - video by Ingonyama. (17.08.2023)
+* [On-Chain Report ep3: Zero Knowledge Cryptography with Brendan Farmer from Polygon](https://www.youtube.com/watch?v=dbRf2ERUTCg) - talk with Brendan Farmer. (18.08.2023)
+* [Tree Types: Inside Aztec Labs](https://www.youtube.com/watch?v=ejZe98ZuiKU) - webinar by Sean Cheetham. (19.08.2023)
```
Comment (zie1ony) [inline newsletters/2023-08-23-newsletter.md:43]: Same link as below?

**Review by zie1ony — APPROVED** [review]
Nice content :) Use top picks I've marked

**Pull request merged**